src/globalInstance.ts

```diff
@@ -10,6 +10,7 @@ export function getGlobalInstance(): Monitor {
     if (!instance) {
         instance = new Monitor({
```

Comment:
won't it be better to have a static instance of "empty" instead of making new each time?

Reply:
I'd say we should throw if the global instance has not been initialized, no?
Also, he's not creating a new one each time, only the first time the method is called before `setGlobalInstance` has been called.

Reply:
Yeah, ok...